packages/cli/src/lib/commands/sequence.ts

```diff
@@ -127,8 +139,21 @@ export const sequence: CommandDefinition = (program) => {
     sequenceCmd
         .command("send")
         .argument("<package>", "The file to upload or '-' to use the last packed")
+        .option("--name <name>", "Allows to name sequence")
         .description("Send the Sequence package to the Hub")
-        .action(async (sequencePackage: string) => sendPackage(sequencePackage, profileConfig.format));
+        .action(
+            async (sequencePackage: string, { name }) => sendPackage(sequencePackage, { name }, profileConfig.format)
+        );
+
+    sequenceCmd
+        .command("update")
+        .argument("<package>", "The file to upload or '-' to use the last packed")
+        .option("--name <name>", "Sequence to be updated (by name)")
```

Comment:
I think this `--name` option suggests changing the name for the sequence rather than the sequence package for the name. Or maybe it would be a good idea to support both options? Like: 
- update the name for the sequence already sent to Hub
- update the seq package under the name (which is already implemented `--name`)